### PR TITLE
Igor/fusedbatchnorm fix

### PR DESCRIPTION
**Pull request by megawidget**
Fixed FusedBatchNorm to actually work.  It also now has a test.  Furthermore, tweaked `GetInputNode` to provide more detail on errors.

The test fails sometimes when rtol < 0.01, which doesn't seem right to me.  Also, I updated the epsilon to TensorFlow's default, let me know if that's the wrong thing to do.

### CODE REVIEW

File: test/python/test_fusedbatchnorm.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+class TestFusedBatchNorm(NgraphTest):
+  x = np.random.rand(64, 3, 10, 8).astype('f')
+  scale = [1.0, 0.9, 1.1]
+  offset = [0.1, 0.2, -.3]
+
+  def test_fusedbatchnorm_nchw(self):
+    with self.device:
+      norm = tf.nn.fused_batch_norm(self.x, self.scale, self.offset,
+                                    data_format='NCHW')
+
```
Comment (sayantan-nervana) [inline test/python/test_fusedbatchnorm.py]: The TranslateOp function of batchnorm has 2 paths one for training one for testing.
By default the 'is_training' parameter of tf.nn.fused_batch_norm is true. We should have a testcase that activates the test path of the TranslateOp with is_training=false

Comment (sayantan-nervana) [reply test/python/test_fusedbatchnorm.py]: Not related to this change directly: tf_is_training in TranslateFusedBatchNormGradOp is being set, but never used. Maybe we can get rid of that?

Comment (sayantan-nervana) [reply test/python/test_fusedbatchnorm.py]: "is_training attribute not present, setting to true" ... Not sure if we prefer true or false. For grad ops we can clearly chose is_trainable to be true, not sure for non-grad ops.

Comment (aprocter) [reply test/python/test_fusedbatchnorm.py]: > "is_training attribute not present, setting to true" ... Not sure if we prefer true or false. For grad ops we can clearly chose is_trainable to be true, not sure for non-grad ops.

I think I wrote that, back when I wasn't sure whether we would need to set the default attribute values ourselves in the builder, or could rely on them to be inserted by TF itself. It appears to be the latter.

Same thing with the `epsilon` attribute.

File: src/ngraph_builder.cc [hunk 2]
```diff
@@ -131,10 +131,19 @@ static tf::Status GetInputNode(const Builder::OpMap& ng_op_map,
 
   tf::Node* tf_input;
   TF_RETURN_IF_ERROR(op->input_node(input_idx, &tf_input));
+  const std::vector<shared_ptr<ng::Node>>* ng_op = nullptr;
```
Comment (aprocter) [inline src/ngraph_builder.cc]: Why does `ng_op` need to be a pointer?

Comment (megawidget) [reply src/ngraph_builder.cc]: because it cannot be a reference if it gets set later.

Comment (aprocter) [reply src/ngraph_builder.cc]: Oooh okay, and that assignment has to be scoped inside the `try` so we can't set it at initialization time without makin' a mess. Works for me.
